backend/src/migration/1665655290096-AddHumanReadableNameToModel.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddHumanReadableNameToModel1665655290096 implements MigrationInterface {
+  name = "AddHumanReadableNameToModel1665655290096";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "model" ADD "humanReadableName" character varying NOT NULL`);
```

Comment:
Migraatio epäonnistuu, koska uusi sarake on `NOT NULL`. Tämän voi korjata vaikka näin ja päivittää oikeat nimet tuotantoonviennin jälkeen:
```suggestion
    await queryRunner.query(`ALTER TABLE "model" ADD "humanReadableName" character varying NULL`);
    await queryRunner.query(`UPDATE "model" SET "humanReadableName" = "id"`);
    await queryRunner.query(`ALTER TABLE "model" ALTER COLUMN "humanReadableName" SET NOT NULL`);
```

Reply:
Äh, ainiin...